mllib/src/test/scala/org/apache/spark/mllib/evaluation/MulticlassMetricsSuite.scala

```diff
@@ -72,6 +72,7 @@ class MulticlassMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(math.abs(metrics.recall -
```

Comment:
Also it would be tidier to change the test to first assert about accuracy, and then assert that every other metric is close to accuracy, rather than to recall. Actually, we should assert exact equality in all of those comparisons to precision, recall etc.